tasks/kmt.py

```diff
@@ -24,6 +26,7 @@
 
 X86_AMI_ID_SANDBOX = "ami-0d1f81cfdbd5b0188"
 ARM_AMI_ID_SANDBOX = "ami-02cb18e91afb3777c"
+GOVERSION = 1.19
```

Comment:
Should be 1.20 now

Reply:
Updating the images as well
https://github.com/DataDog/ami-builder/pull/109